.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: rust
+rust:
+- nightly-2016-04-11
+before_script:
+- |
+  pip install 'travis-cargo<0.2' --user &&
```

Comment:
Wow, it works! Should I add `--nocapture` so we can see the actual test
output even if there was no failure?

I'll add nightly as soon as I get to a computer. So you want to hate the
build on this or should it be an allowed failure?

Tbh, I mostly copied the Travis file from my other projects, which all use
more of travis-cargos's features. travis-cargo is pretty nice, you can e.g.
easily generate an upload documentation (needs a github token with access
to the repo) or run coverage tests.
Scott Olson <notifications@github.com> schrieb am Fr., 15. Apr. 2016 um
05:25:

> In .travis.yml <https://github.com/tsion/miri/pull/3#discussion_r59823208>
> :
>
> > @@ -0,0 +1,14 @@
> > +language: rust
> > +rust:
> > +- nightly-2016-04-11
> > +before_script:
> > +- |
> > +  pip install 'travis-cargo<0.2' --user &&
>
> @killercup <https://github.com/killercup> I glanced at travis-cargo's
> readme, but I'm still unsure what benefit it provides over running cargo
> directly in this case. Can you explain?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tsion/miri/pull/3/files/99767e7a293c2ea26725a047fcc870af97d58ee3#r59823208>
>